index.html

```diff
@@ -15,8 +15,8 @@
                 <h1 class="title white">
                     Hello Github!
                 </h1>
-                <p>your-name: <strong>NAMEtest</strong></p>
-                <p>your-age:<strong>AGEtest</strong></p>
+                <p>nethmi sahasrika <strong>NAMEtest</strong></p>
+                <p>your20<strong>AGEtest</strong></p>
```

Comment:
Appreciate your contribution! Please remove the temporary text as well... :sparkles: